### PR TITLE
Fix 516: 

### PR DESCRIPTION
**Pull request by shimono**

### CODE REVIEW

File: src/test/java/io/personium/core/rs/cell/TokenEndPointResourceTest.java [hunk 1]
```diff
@@ -516,9 +532,103 @@ public void token_GrantCode_WithConfidentialMark_When_Valid_Succeeds() throws Ex
...
+        List<Role> roleList = new ArrayList<Role>();
+        roleList.add(new Role("confidentialClient", null, null, clientId));
+        TransCellAccessToken appAuthToken = new TransCellAccessToken(clientId, clientId + "#app",
+                this.mockCell.getUrl(), roleList, "", new String[0]);
+
+        // prepare TCAT
+        String issuerCellUrl = this.mockCell.getUnitUrl() + "issuerCell/";
+        roleList = new ArrayList<Role>();
+        Role role1 = new Role("MyBoardEditor", "mb", this.mockCell.getUnitUrl() + "appcell/", issuerCellUrl);
+        Role role2 = new Role("MyBoardViewer", "mb", this.mockCell.getUnitUrl() + "appcell/", issuerCellUrl);
```
Comment (tochi-y) [inline src/test/java/io/personium/core/rs/cell/TokenEndPointResourceTest.java]: Better to use clientId.

File: src/test/java/io/personium/core/rs/cell/TokenEndPointResourceTest.java [hunk 2]
```diff
@@ -516,9 +532,103 @@ public void token_GrantCode_WithConfidentialMark_When_Valid_Succeeds() throws Ex
...
+        String tokenStr = json.getString("refresh_token");
+        vrt = (VisitorRefreshToken) AbstractOAuth2Token.parse(tokenStr, this.mockCell.getUrl(),
+                this.mockCell.getUnitUrl());
+        for (Role role : vrt.getRoleList()) {
+            log.info(role.toRoleClassURL());
+        }
+        assertEquals(role1.toRoleClassURL(),
+                vrt.getRoleList().get(0).toRoleClassURL());
+        assertEquals(role2.toRoleClassURL(),
+                vrt.getRoleList().get(1).toRoleClassURL());
```
Comment (tochi-y) [inline src/test/java/io/personium/core/rs/cell/TokenEndPointResourceTest.java]: To assert VisitorAccessTokken is necessary.

File: src/test/java/io/personium/core/rs/cell/TokenEndPointResourceTest.java (hunk 1 above)
Comment (shimono) [reply src/test/java/io/personium/core/rs/cell/TokenEndPointResourceTest.java]: Yes

Comment (shimono) [reply src/test/java/io/personium/core/rs/cell/TokenEndPointResourceTest.java]: Fixed

File: src/test/java/io/personium/core/rs/cell/TokenEndPointResourceTest.java (hunk 2 above)
Comment (shimono) [reply src/test/java/io/personium/core/rs/cell/TokenEndPointResourceTest.java]: Fixed

**Pull request merged**